.swm/doc-templates/how-to-add-something.92m45.t.sw.md

```diff
@@ -0,0 +1,38 @@
+---
+id: 92m45
+name: How to add something
+file_version: 1.0.2
+app_version: 0.7.1-1
+---
+
+## Introduction
+
+In this document, we will learn how to add a new {{SOMETHING (e.g., a component, a class, a plugin)}} to the system.
+
+{{SOMETHING}} is {{explain its role in the system}}.
+
+Some examples of {{SOMETHING}}s are {{examples}}.`
+
+## Basic example
+
+We'll follow the implementation of {{EXAMPLE (an instance of the SOMETHING, e.g., a sub-class)}} for this example.
+
+Every time we add a new instance, we follow these steps:
+<br/>
+<!-- TEMPLATE-swimm-snippet-placeholder -->
+Add a snippet of the first step needed to implement the EXAMPLE
```

Comment:
Instructions. {{Add a snippet of the first step needed to implement the EXAMPLE}}

Reply:
@nadav-swimm this is within the snippet placeholder - so it appears within the placeholder.. I think we shouldn't use {{ for these...